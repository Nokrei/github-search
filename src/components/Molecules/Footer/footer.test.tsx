import { Footer } from ".";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";

describe("Footer", () => {
  it("renders a footer", () => {
    render(<Footer />);
    expect(
      screen.getByText("Created by Piotr Mrozowski for KMMRCE")
    ).toBeInTheDocument();
  });

  it("snapshot test", () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
