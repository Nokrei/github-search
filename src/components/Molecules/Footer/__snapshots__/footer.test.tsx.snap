// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test 1`] = `
<footer
  class="bg-purple-900 p-2 text-center"
>
  <p
    class="text-white"
  >
    Created by Piotr Mrozowski for KMMRCE
  </p>
</footer>
`;
